fxgl-events/src/main/kotlin/com/almasb/fxgl/event/EventBus.kt

```diff
@@ -24,6 +24,8 @@ class EventBus {
 
     private val eventHandlers = Group()
 
+    private var loggingEnabled = true
```

Comment:
1. Please remove `private`, Kotlin will auto-generate `getter/setter` pair, so the two new functions below are no longer needed.
2. Please rename variable name to `isLoggingEnabled` to preserve consistency in the codebase (all boolean variables begin with `is`).

Reply:
Fixed